cfme/cloud/flavor.py

```diff
@@ -97,25 +96,88 @@ def is_displayed(self):
 
     breadcrumb = BreadCrumb()
     title = Text('//div[@id="main-content"]//h3')
-    entities = View.nested(BaseNonInteractiveEntitiesView)
+    category = BootstrapSelect("tag_cat")
+    tag = BootstrapSelect("tag_add")
     save = Button('Save')
     reset = Button('Reset')
     cancel = Button('Cancel')
 
 
+class FlavorCollection(Navigatable):
```

Comment:
We should use `NavigatableMixin` now for the `appliance` thing

Reply:
Even more, we should be using BaseCollection.

May want to hold work on this PR in general until Collections implementation is more established, and focus on WT + test fixes.